CoastalResilienceNetwork/GeositeFramework
# Style fixes

#### Overview
A few straggling UI updates:
* Icon size for Launchpad was too large.  It's resized now to the smaller size of other apps, and the whole set it enlarged to its intended size
* Make slider styles specific to `.slider-container` to remove conflicts with `ui-state-default` elements.
* Add grunt project to `styleguide`: allows us to make edits to `styleguide` and build css assets to include in `css/`
* Position absolute positioned plugin elements within sidebar container

Partially connects #829 
Partially connects https://github.com/CoastalResilienceNetwork/regional-planning/issues/71

#### Testing
* Verify that the icons are of similar size:
![screenshot from 2017-01-09 20 56 34](https://cloud.githubusercontent.com/assets/1014341/21791028/26c848f6-d6ae-11e6-85e8-b92f321b0008.png)
* Clone [a plugin that uses jQuery UI](https://github.com/CoastalResilienceNetwork/benefits_explorer_prototype) and verify it renders correctly.
![screenshot from 2017-01-09 21 14 29](https://cloud.githubusercontent.com/assets/1014341/21791419/a39fcde8-d6b0-11e6-94f9-7b7d424d9039.png)
* From `styleguide`, run `npm install` and `grunt` and verify you can build and view the styleguide subproject
* Open Regional Planning and verify the spinner is contained in the plugin and not overlaying the whole screen on startup
